### PR TITLE
Document Slave's DNS config [Issue #79].

### PR DESCRIPTION
**Pull request by dhananjay92**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -63,6 +63,24 @@ Data Collector
 --------------
 Drop `website/UPDATE.PHP` and `website/internals.php` somewhere, and rename `UPDATE.PHP` to something secret. Make sure you don't have directory listings enabled.
 
+Slave DNS Config
+----------------
+Before running the benchmarks, add these lines to your `/etc/hosts` file and flush DNS cache. These host addresses are used by `benchmarks_remote.py`. This configuration is needed for all the slaves.
+
```
Comment (h4writer) [inline README.md]: that use the remote or local benchmarks. Only the shell benchmarks don't need it.

Comment (dhananjay92) [inline README.md]: Added that. Done.

**Comment by dhananjay92** [comment]
@h4writer: Just one more thing. What does all three values in [execute.py#L49](https://github.com/h4writer/arewefastyet/blob/master/slave/execute.py#L49) mean? For servo, I need to add one more mode there.

**Comment by h4writer** [comment]
When sending the data to the server (RemoteSubmitter), we have information about the machine, the engine and information about the config. But we need to know which line (mode) on the graph this data belongs too. That is what this does:
 "firefox,default:jmim",
Engine: Firefox
Config: Default
=> send the data as Mode jmim

On the line you specified are the default modes. But it can get adjusted using the commandline. E.g.
https://arewefastyet.com/#machine=31
This machine is a nice example of non-default modes.

This graph has 4 modes:
    Firefox (OSX) == mozilla_osx
    Firefox (Windows 8) == mozilla_win8
    Firefox (Ubuntu) == mozilla_ubuntu
    Firefox (Windows 10) == mozilla_win10

But all the 4 lines are running engine: "Firefox" and config "Default".
To distinguish between the modes, we do

python execute.py --submitter-mode firefox,default:mozilla_osx
python execute.py --submitter-mode firefox,default:mozilla_win8
python execute.py --submitter-mode firefox,default:mozilla_ubuntu
python execute.py --submitter-mode firefox,default:mozilla_win10

on the right machine. (To override the default mode)

I think you can just add "servo,default:servo"

**Comment by dhananjay92** [comment]
Okay, Thanks!

**Pull request merged**